egs/ami/s5b/local/chain/tuning/run_tdnn_1b.sh

```diff
@@ -212,9 +212,9 @@ if [ $stage -le 16 ]; then
      /export/b0{5,6,7,8}/$USER/kaldi-data/egs/ami-$(date +'%m_%d_%H_%M')/s5b/$dir/egs/storage $dir/egs/storage
   fi
 
- touch $dir/egs/.nodelete # keep egs around when that run dies.
+  mkdir -p $dir/egs && touch $dir/egs/.nodelete # keep egs around when that run dies.
```

Comment:
Actually, would you mind just deleting these lines?  The 'proper' way now to stop the egs being removed is to run with --cleanup false.  Anyway it isn't necessarily a good default to keep the egs around.